tests/kernel/semaphore/semaphore/src/main.c

```diff
@@ -553,8 +613,83 @@ void test_sem_take_multiple(void)
 
 	signal_count = k_sem_count_get(&low_prio_sem);
 	zassert_true(signal_count == 1U,
-		     "low priority thread didn't get executed");
+			"Low priority thread don't get the sem");
```

Comment:
Please rephrase - "Low priority thread did not get the sem"

Reply:
Thank you, you are not only review the code, but also help to improve the grammar. I appreciate that^_^

Reply:
did not get the semaphore, do not use sem instead of semaphore in documentation and comments and anything that not code.

Reply:
I replaced everywhere didn't -> did not.